scripts/docker-compose-prod.yaml

```diff
@@ -0,0 +1,96 @@
+services:
+
+  db:
+    container_name: ${POSTGRES_CONTAINER_NAME}
+    image: postgres
+    environment:
+      #      ENVIRONMENT: ${APP_ENV}
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DATABASE}
+    volumes:
+      - ../data/postgres:/var/lib/postgresql/data
+    ports:
+      - "${DB_PORT}:${POSTGRES_PORT}"
+    networks:
+      - database_network
+
+  backend:
+    container_name: "${BACKEND_CONTAINER_NAME}"
+    image: "${BACKEND_IMAGE_NAME}"
+    environment:
+      ENVIRONMENT: ${APP_ENV}
+      CONTAINER_NAME: ${BACKEND_CONTAINER_NAME}
+      DB_HOST: db
+      DB_USER: ${POSTGRES_USER}
+      DB_PASS: ${POSTGRES_PASSWORD}
+    env_file:
+      - ../.env
+    ports:
+      - "${BACKEND_HOSTPORT}:${BACKEND_PORT}"
+    volumes:
+      - ../backend:/backend
+      - ../.env:/backend/.env.dev
+    command: reflex -r '\.go$' -s -- sh -c './supervisor.sh'
+    tty: true
+    networks:
+      - database_network
+      - frontend_network
+    depends_on:
+      - db
+
+  frontend:
+    container_name: "${FRONTEND_CONTAINER_NAME}"
+    image: "${FRONTEND_IMAGE_NAME}"
+    environment:
+      ENVIRONMENT: ${APP_ENV}
+      DOMAIN: "https://${DOMAIN}"
+    networks:
+      - frontend_network
+    env_file:
+      - ../.env
+    volumes:
+      - ../frontend:/app
+      - /app/node_modules
+    stdin_open: true
+    ports:
+      - "${REACT_HOSTPORT}:${REACT_PORT}"
+    depends_on:
+      - backend
+
+  webserver:
+    container_name: "${NGINX_CONTAINER_NAME}"
+    image: nginx:latest
+    environment:
+      ENVIRONMENT: ${APP_ENV}
+      DOMAIN: ${DOMAIN}
+      BACKEND_PORT: ${BACKEND_PORT}
+      FRONTEND_PORT: ${REACT_PORT}
+    ports:
+      - 80:80
+      - 443:443
+    volumes:
+      - ../data/nginx/vhosts:/etc/nginx/templates/
+      - ../data/certbot/conf:/etc/letsencrypt
+      - ../data/certbot/www:/var/www/certbot
+    restart: always
+    networks:
+      - database_network
+      - frontend_network
+    depends_on:
+      - backend
+      - frontend
+
+  certbot:
+    image: certbot/certbot:latest
```

Comment:
@ntoufoudis so when I do, "docker ps -a" I get the following for the certbot image name:

```
scripts-certbot-1
```

Do you think we should instead provide a name for this, e.g. `socialpredict-certbot` so that we can differentiate it and so it's more specific?

Reply:
Commit [3dbb969](https://github.com/openpredictionmarkets/socialpredict/commit/3dbb969329ff746de5ec614e951cba0e2ef3608f) fixes this issue.